Tests/Unit/Integration/MultipleItemsProcFuncTest.php

```diff
@@ -31,10 +32,12 @@ public function testRegistersFunction(): void
 
     public function testExecutesFunction(): void
     {
-        $provider = $this->getMockBuilder(FormDataProviderInterface::class)->getMockForAbstractClass();
+        $formDataProviderInterface = $this->getMockBuilder(FormDataProviderInterface::class)->getMockForAbstractClass();
```

Comment:
If one of these classes only exists on some versions of TYPO3, then this test method should be split in two and a check made for `class_exists` and if it doesn't, do `$this->markTestSkipped('Skipped on v12');` so the test won't run and always fail on that TYPO3 version.